[16.0][MIG] sale_order_lot_generator

florian-dacosta opened the pull request:
depends on https://github.com/OCA/sale-workflow/pull/2282

mourad-ehm committed:
[MIG] sale_order_lot_generator: Migrated to 10.0

mourad-ehm committed:
[FIX] manage lot generation for order lines created after confirmation

OCA-git-bot committed:
[ADD] icon.png

bealdav committed:
IMP black format

bealdav committed:
MIGR 12.0 sale_order_lot_generator

OCA-git-bot committed:
sale_order_lot_generator 12.0.1.0.1

florian-dacosta committed:
[IMP] sale_order_lot_generator: black, isort, prettier

florian-dacosta committed:
[MIG] sale_order_lot_generator: Migration to 14.0

Commit:
[FIX] allow to confirm several SOs at once

oca-travis committed:
[UPD] Update sale_order_lot_generator.pot

OCA-git-bot committed:
[UPD] README.rst

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: sale-workflow-14.0/sale-workflow-14.0-sale_order_lot_generator
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-14-0/sale-workflow-14-0-sale_order_lot_generator/

newtratip committed:
[14.0][FIX] sale_order_lot_generator

danimv5 committed:
Translated using Weblate (Catalan)

Currently translated at 100.0% (8 of 8 strings)

Translation: sale-workflow-14.0/sale-workflow-14.0-sale_order_lot_generator
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-14-0/sale-workflow-14-0-sale_order_lot_generator/ca/

danimv5 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (8 of 8 strings)

Translation: sale-workflow-14.0/sale-workflow-14.0-sale_order_lot_generator
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-14-0/sale-workflow-14-0-sale_order_lot_generator/es/

OCA-git-bot committed:
sale_order_lot_generator 14.0.1.0.1

Commit:
[FIX] allow to create lines in redrafted orders

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 12.5% (1 of 8 strings)

Translation: sale-workflow-14.0/sale-workflow-14.0-sale_order_lot_generator
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-14-0/sale-workflow-14-0-sale_order_lot_generator/it/

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[UPD] README.rst

florian-dacosta committed:
[IMP] sale_order_lot_generator: black, isort, prettier

florian-dacosta committed:
[MIG][16.0] Migrate sale_order_lot_generator to v16

rousseldenis commented:
/ocabot migration sale_order_lot_generator

florian-dacosta commented:
@bealdav @sebastienbeau @newtratip  Ready for review

bealdav commented on sale_order_lot_generator/models/sale_order_line.py:
```diff
@@ -21,21 +21,21 @@ def _prepare_vals_lot_number(self, index_lot):
     def create_prodlot(self, index_lot=1):
```
To be consistent I supose create_prodlot should be rename without prod

florian-dacosta replied:
Done

florian-dacosta replied:
Done

rvalyi replied:
a detail, but if you changed create_prodlot to create_lot, I would change also the generate_prodlot method to generate_lot. If you do it I approve right now...

florian-dacosta replied:
Done

florian-dacosta commented:
@bealdav @rvalyi changes have been done, feel free to update your reviews!

florian-dacosta commented:
PR is quite old and approved, can any PSC merge this one ?
@sebastienbeau @rvalyi @rousseldenis

rvalyi commented:
/ocabot merge nobump

florian-dacosta committed:
[FIX] Stop deleting lot on sale order cancelation
A lot may have happen, and it is possible to have done some MO, received some PO using this lot, maybe there is even stock on hand with this lot and should not delete it or it is lost everywhere.

bguillot commented:
/ocabot merge nobump
